src/api/spec/features/webui/workflow_runs_filter_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'browser_helper'
+
+RSpec.describe 'Workflow runs filters' do
```

Comment:
You should test the view components and filter class directly instead of firing up a browser to do this...

Reply:
Don't get me wrong, I have nothing against testing the happy path of this feature but how it behaves should be tested as close to where the code is...

Reply:
Maybe testing simple components here is a bit of an effort that vanishes soon, as discussed in https://github.com/openSUSE/open-build-service/pull/15950#discussion_r1562364224 . No?

Reply:
I did not mean "test every component". I mean test that this feature behaves correctly as close to the code as you can. That is usually *not* the browser. My guess is this is currently: `WorkflowRunFinder`, `WorkflowRunController` and `WorkflowRunFilterComponent`.